src/components/Admin/Users.js

```diff
@@ -5,7 +5,7 @@ import { Button, Table, Tag, notification, Input, Icon, Alert } from 'antd';
 import Highlighter from 'react-highlight-words';
 
 import { getAllUsers, deleteUser, deleteAllServices } from "../../redux/actions/adminActions";
-import './users.css';
+import './Users.css';
```

Comment:
File Name is `users.css` not `Users.css`
This is why travis is failing

Reply:
@harsh-jindal but I changed the file name to Users.css to keep it consistent with other files names.

Reply:
@ashwaniYDV PLease check in files changed what files have been changed. You have not changed `users.css` to `Users.css`

Reply:
@harsh-jindal I have changed Users.css to users.css still Travis is failing. Please help. There is no such error in my console